pkg/routeagent_driver/handlers/ovn/handler.go

```diff
@@ -85,7 +98,49 @@ func (ovn *Handler) Init() error {
 
 	ovn.startRouteConfigSyncer(ovn.stopCh)
 
-	return ovn.ensureSubmarinerNodeBridge()
+	connectionHandler := NewConnectionHandler(ovn.k8sClientset)
+
+	err = connectionHandler.initClients()
+	if err != nil {
+		return errors.Wrapf(err, "error getting connection handler to connect to OvnDB")
+	}
+
+	if ovn.watcherConfig.RestMapper == nil {
+		if ovn.watcherConfig.RestMapper, err = admUtil.BuildRestMapper(ovn.watcherConfig.RestConfig); err != nil {
+			return errors.Wrap(err, "error creating the RestMapper")
+		}
+	}
+
+	if ovn.watcherConfig.Client == nil {
+		if ovn.watcherConfig.Client, err = dynamic.NewForConfig(ovn.watcherConfig.RestConfig); err != nil {
+			return errors.Wrap(err, "error creating dynamic client")
+		}
+	}
+
+	if ovn.watcherConfig.Scheme == nil {
+		ovn.watcherConfig.Scheme = scheme.Scheme
+	}
```

Comment:
You don't need to initialize these - `watcher.New` will do that.

```suggestion
```

Reply:
Done.